<launch>
<node
pkg="agitr"
type="lane_track"
name="lane_track"
output="screen"
/>
<node
pkg="agitr"
type="ACC_control"
name="ACC_control"
output="screen"
/>
<node
pkg="agitr"
type="fusion"
name="fusion"
output="screen"
/>
<node
pkg="agitr"
type="debuge_rviz"
name="debuge_rviz"
output="screen"
/>
</launch>
